src/lib/tooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { createRoot } from 'react-dom/client'
+import { TooltipBlock } from '../components/TooltipBlock'
+import glossary from '../../data/glossary-tooltip.json'
+
+const OS_RAISE_IB_TOOLTIP_CLASS = 'os-raise-ib-tooltip'
+
+const glossaryLookup = (key: string): string | undefined => {
+  const glossaryMap = new Map<string, string>(Object.entries(glossary).map(entry => [entry[0].toLowerCase(), entry[1]]))
+
+  return glossaryMap.get(key)
+}
+
+export const tooltipify = (element: HTMLElement): void => {
```

Comment:
I think codecov is probably dinging you due to lack of coverage on this function. One way you could write a unit test for this is by following [this pattern](https://github.com/openstax/k12-apps-raise/blob/5e04bd15a7533c02139139eabf1230c5e06c7650/src/tests/blocks.test.ts#L85). The one gotcha is at least I was unable to find a way to get that mock for `createRoot` to work easily with `<React.StrictMode>` so you may have to forego that here if you go that route :man_shrugging: (though if you find a way to do it without that LMK!)